frontends/bmd/src/pages/admin_screen.tsx

```diff
@@ -198,17 +198,17 @@ export function AdminScreen({
           {election ? (
             <p>
               <Text as="span" voteIcon>
-                Election definition is loaded.
+                Election Definition is loaded.
```

Comment:
Got confirmation from @beausmith that we should consistently title case "Election Definition" in VxMark (we currently use a mix of title case and sentence case)